setup.py

```diff
@@ -5,7 +5,8 @@
 """
 from abc import abstractmethod
 # Disabling checks due to https://github.com/PyCQA/pylint/issues/73
-from distutils.command.clean import clean  # pylint: disable=E0401,E0611
+# pylint: disable=import-error,no-name-in-module
```

Comment:
This should not affect the rest of the module. Either write in the same lame as `from` (recommended) or enable it right after the import.

Reply:
FIxed.

Reply:
@macartur, the way it is, in a line of its own and with no indentation, these errors are disabled from line 8 to the end of the file. They should be disabled only in one line.